test/zepto.html

```diff
@@ -2664,11 +2670,16 @@
             })
           })
         })
-        t.assertTrue(fired)
-        t.assertIdentical(Zepto, arg1)
-        t.assertIdentical(Zepto, arg2)
-        t.assertIdentical(Zepto, arg3)
-        t.assertIdentical(Zepto, arg4)
+        t.pause()
+        setTimeout(function(){
+          t.resume(function(){
+            t.assertTrue(fired)
+            t.assertIdentical(Zepto, arg1)
+            t.assertIdentical(Zepto, arg2)
+            t.assertIdentical(Zepto, arg3)
+            t.assertIdentical(Zepto, arg4)
+          })
+        }, 500)
```

Comment:
Being written that way the test just checks `fired` and `arg[1234]` values and does nothing around fact that a ready callback should be called soon. Sounds good. Since that I would be change comment to

> // Check that ready() callback fired and get Zepto as an argument

It also worth to remove `fired` as it becomes redundant.